cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -705,6 +705,16 @@
     }
 
     /**
+     * Secure Store
+     */
+    public static final class Store {
+      /** Location of the secure store file. /tmp will be used if none is provided.*/
+      public static final String SECURE_STORE_FILE_PATH = "security.store.file.path";
```

Comment:
by convention, just call these as ``FILE_PATH`` and ``FILE_PASSWORD``. So they can be referred as Security.Store.FILE_PATH and Security.Store.FILE_PASSWORD.